src/base/AsyncJob.cc

```diff
@@ -199,15 +197,14 @@ AsyncJob::ReportAllJobs(StoreEntry *e)
 {
     PackableStream os(*e);
     // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
-    const char *indent = "    ";
+    const std::string indent("  ");
     for (const auto job: AllJobs()) {
-        os << indent << job->id << ":\n";
-        os << indent << indent << "type: '" << job->typeName << "'\n";
-        os << indent << indent << "status:" << job->status() << '\n';
-        if (!job->started_)
-            os << indent << indent << "started: false\n";
-        if (job->stopReason)
-            os << indent << indent << "stopped: '" << job->stopReason << "'\n";
```

Comment:
Please continue to report the reason here. I am aware that it will also be reported by default status() implementation (that problem needs a different fix).

Reply:
done